dynamodb_python/table.py

```diff
@@ -0,0 +1,428 @@
+import datetime
```

Comment:
`key` and `time` our internal primary key for DynamoDB. It can vary from one Table to another. Also, their type can be not the `str` and `int` respectively, so this should be considered, as well.